src/PhysicsSystem.cpp

```diff
@@ -0,0 +1,9 @@
+#include "PhysicsSystem.h"
+
+PhysicsSystem::PhysicsSystem()
+    : gFoundation_(PxCreateFoundation(PX_FOUNDATION_VERSION, gAllocator_, gErrorCallback_)) {
+}
+
+PhysicsSystem::~PhysicsSystem() {
```

Comment:
Is there any reason to be explicit with the deconstructor? Just curious, not blocking for this.

Reply:
I just always feel like explicit is better. Sometimes it's more ok if the destructor will probably not be used. But I feel like hte PhysicsSystem might have some deinit calls to PHYSX